### PR TITLE
Split one CustomPainter doc paragraph to bullet points

### PR DESCRIPTION
**Pull request by xster**
The sentence was too long. I had to read it a few times to understand.

### CODE REVIEW

**Comment by xster** [comment]
Also added one line to say shouldRepaint's param is never null so users don't have to guess. Looking at implementation, seems like it would crash if it were ever null anyway.

File: packages/flutter/lib/src/rendering/custom_paint.dart [hunk 1]
```diff
@@ -234,6 +238,8 @@ abstract class CustomPainter extends Listenable {
   /// repaints should be avoided as much as possible, a [RepaintBoundary] or
   /// [RenderRepaintBoundary] (or other render object with
   /// [RenderObject.isRepaintBoundary] set to true) might be helpful.
+  ///
+  /// `oldDelegate` is never null.
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/custom_paint.dart]: Avoid starting sentences with a lower case letter, e.g. "The `oldDelegate` argument will never be null."

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/rendering/custom_paint.dart (hunk 1 above)
Comment (xster) [reply packages/flutter/lib/src/rendering/custom_paint.dart]: Oops. We should add a linter.

**Pull request merged**